Source/Widgets/Viewer/Viewer.js

```diff
@@ -947,7 +947,7 @@ Object.defineProperties(Viewer.prototype, {
    * Gets the DOM element for the area at the bottom of the window containing the
    * {@link CreditDisplay} and potentially other things.
    * @memberof Viewer.prototype
-   * @type {Element}
+   * @type {HTMLDivElement}
```

Comment:
Is there reason to use `HTMLDivElement` instead of `HTMLElement`? `HTMLElement` is more generic and has the style property as well.

Reply:
From the actual code, it is indeed an HTMLDivElement.

![bottomContainer](https://puxiao.com/temp/bottomcontainer.jpg)



But if you want to define it as HtmlElement, it doesn't seem to be a problem.

You decide.

Reply:
I see that viewer.infoBox.container is currently also an Element. 
If the user does not define it, it will be created as HTMLDivElement internally.

At least, I think all Element should be replaced with HTMLElement.